.themes/default/source/arcs.html

```diff
@@ -2,20 +2,32 @@
 layout: page
 title: Arc Index
 rp_arcs: true
----
-
+--- 
+    
 <ul class="rp-list">
 {% for x in page['rps'] %}
   {% assign type = x[0] %}
   {% assign rp = x[1] %}
   {% if type == 'arc' %}
     <h3 class="arc-title"><a name="{{ rp.name | slugify }}" href="#{{ rp.name | slugify }}">{{ rp.name }}</a></h3>
+    <p class="arc-description">
+    {% for arc_rp in rp.rps %}
```

Comment:
How so? Given that one could just as easily ignore the feature in favor of using arcs? But the whole idea of a canon is that X stories fit into it, why Y stories fit into a different one. If the same story fits into X and Y then X and Y are the same canon.